src/index.ts

```diff
@@ -340,10 +339,10 @@ class Logging {
       name: string, config: CreateSinkRequest,
       callback?: CreateSinkCallback): Promise<[Sink, LogSink]>|void {
     const self = this;
-    if (!is.string(name)) {
+    if (typeof name !== 'string') {
```

Comment:
Would this be considered semver major?

[rant] Removal of `is` as a dependency doesn't fit into the spirit of this PR anyway. This only makes the review process more complex. Refactor PRs should be easy to review.

Open a separate PR for this change.
[/rant]

Reply:
Happy to :) So question tho - would you prefer a total drop of `is`, a move to the other `is`, or keeping this as `is` 🥁

Reply:
No objections to dropping `is` as semver-major. If the argument is that it is not breaking, it needs to be made.